packages/earth-map/src/components/sidebar/component.tsx

```diff
@@ -44,32 +76,32 @@ class SidebarComponent extends React.Component<ISidebar> {
     }
 
     return (
-      <Sidebar native state={state}>
-        {({ x, opacity, ...props }) => (
+      <SidebarPanel native state={state}>
+        {({ x, ...props }) => (
           <animated.div
             ref={(r) => {
               this.sidebarPanel = r;
             }}
-            className="c-sidebar"
+            className={classNames({
+              'c-sidebar': true,
+              'ng-c-sidebar': true,
+              'ng-subsection-background': true,
+              'no-scroll': layersPanel,
+            })}
             style={{
-              transform: x.interpolate((x) => `translate3d(${x}px,0,0)`),
+              transform: x.interpolate((x) => `translate3d(${x}%,0,0)`),
               ...props,
             }}
           >
-            <button type="button" onClick={() => this.onClickItem()} className="sidebar--btn">
-              <i
-                className={classNames({
-                  'ng-body-color ng-text-bold ng-icon-small': true,
-                  'ng-icon-directionleft': open,
-                  'ng-icon-directionright': !open,
-                })}
-              />
-            </button>
+            <SidebarClose
```

Comment:
SidebarToggleButton? cause if opens/closes sidebar

Reply:
Will rename it to `<SidebarToggle/>`